jmbase/jmbase/support.py

```diff
@@ -59,7 +60,7 @@ def debug_dump_object(obj, skip_fields=None):
     if skip_fields is None:
         skip_fields = []
     log.debug('Class debug dump, name:' + obj.__class__.__name__)
-    for k, v in obj.__dict__.iteritems():
+    for k, v in iteritems(obj.__dict__):
```

Comment:
Just leaving a note here that `debug_dump_object` is no longer used anywhere, so it and its test can be ... dumped :)

(I'm not asking for it to be removed here, but you can if you want.)